src/popup/router/pages/Names/Claim.vue

```diff
@@ -5,7 +5,12 @@
     </span>
     <InputField
       v-model="name"
-      v-validate="'required|name|name_unregistered'"
+      v-validate="{
+        required: true,
+        name: true,
+        name_unregistered: true,
+        enough_ae: (+getFee + +nameFee).toString(),
```

Comment:
Maybe we need to multiply `getFee` by two, cause we actually send two transactions, but it still wouldn't be accurate enough.

Reply:
Maybe this two fees is not the same in this case we can't multiply or we should agree to add some extra space to cover this case.